src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -501,6 +501,7 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\MapPresenter\MapControl.xaml">
```

Comment:
There's a bad merge or something here makes the build fail :)
@GuillaumeSE

Reply:
Looks like I didn't merge properly. Have fixed it for you @GuillaumeSE.

Reply:
Thanks Will do that!